eternalcore-core/src/main/java/com/eternalcode/core/feature/home/HomeCommand.java

```diff
@@ -59,7 +65,7 @@ void execute(Player player, @Arg Home home) {
         this.teleportToHome(player, home);
     }
 
-    private void teleportToHome(Player player, Home home) {
+    private void teleportToHome(Player player, @Arg Home home) {
```

Comment:
? 
This is not a command
```suggestion
    private void teleportToHome(Player player, Home home) {
```